apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -203,14 +202,14 @@ def to_anndata(
     obsm_varm_width_hints: Optional[Dict[str, Dict[str, int]]] = None,
     uns_keys: Optional[Sequence[str]] = None,
 ) -> ad.AnnData:
-    """Converts the experiment group to `AnnData <https://anndata.readthedocs.io/>`_
-    format. Choice of matrix formats is following what we often see in input
-    ``.h5ad`` files:
+    """Converts the experiment group to AnnData format.
+
+    The choice of matrix formats is following what we often see in input ``.h5ad`` files:
```

Comment:
I really love the idea of linking to another package. Sadly, in the first-sentence-only summaries that Sphinx loves to create, it somehow can't handle these and leaves the summary _entirely blank_. Unacceptable.